easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.5.0-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'libjpeg-turbo'
+version = '1.5.0'
+
+homepage = 'http://sourceforge.net/projects/libjpeg-turbo/'
+description = """libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to accelerate baseline JPEG
+compression and decompression. libjpeg is a library that implements JPEG image encoding, decoding and transcoding.
+"""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('NASM', '2.12.02'),
```

Comment:
NASM 2.13.01 is already installed on daint (but also a more recent version of libjpeg, that would eliminate the need of this dependency).

Reply:
Used libjpeg 1.5.2, removed libjpeg 1.5.0 and NASM 2.12.02 from MR.